src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -193,7 +193,7 @@ public void VerifyTypeIL(string typeName, string expected)
                     Assert.True(found, "Could not find type named " + typeName);
                 }
             }
-            AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output.ToString(), escapeQuotes: false);
+            AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output.ToString().Replace("\t", "    "), escapeQuotes: false);
```

Comment:
VerifyTypeIL was printing the baseline with tabs, which differs from our standard spaces.